src/Makefile

```diff
@@ -318,7 +346,7 @@ endef
 
 # Build the module...
 $(TARGET): $(MODULE_DEPS)
-	$(REDISGRAPH_CC) -o $@ $(CC_OBJECTS) $(LIBS) $(SHOBJ_LDFLAGS) -lc -lm
+	$(REDISGRAPH_CC) -o $@ $(CC_OBJECTS) $(LIBS) $(SHOBJ_LDFLAGS) -lc -lm -lstdc++
```

Comment:
Why was `-lstdc++` added?

Reply:
The icu4c library requires c++ library even for its c api.

Reply:
@stevealexr we don't want to add C++ dependencies to RedisGraph. This is why we requested "minimal subset of the latest ICU library, converted from C++ to C".

Reply:
I will try to convert relevant functions and their dependencies to C.